src/react_native/blur.cljs

```diff
@@ -1,11 +1,8 @@
 (ns react-native.blur
   (:require
     ["@react-native-community/blur" :as blur]
-    [react-native.core :as rn]
-    [react-native.platform :as platform]
     [reagent.core :as reagent]))
 
-(def view (reagent/adapt-react-class (.-BlurView blur)))
-
-;; blur view is currently not working correctly on Android.
-(def ios-view (if platform/ios? view rn/view))
+(def view
+  (when blur
```

Comment:
but view won't be used if blur is nil, it will happen only on Android, and on Android view is not used, but we could remove this for now, idk how this lib will be excluded for Android